defaults/main.yml

```diff
@@ -548,6 +548,10 @@ maas_pip_container_packages:
 
 maas_source_plugin_dir: plugins/
 
+# NOTE: (alextricity25) This variable is used by verify-maas.yml playbook
+# in the RPCO integrated repository.
+maas_rpc_scripts_dir: /opt/rpc-openstack/scripts
```

Comment:
This is not a variable used in this role so it has no reason to be here. IMO this should be renamed within the rpc-o repo as it is used there.